### ISSUE TITLE
ChatGPT Rarely communicates in the chosen language.

### ISSUE THREAD

**Issue by GolfinhoVoador** [YOUSY0US3F/minecraft-gpt-god-plugin#54]
We made a stream using the plugin and ChatGPT did not use Portuguese (as selected on the config) to communicate with the players.

**Comment by GolfinhoVoador** [comment]
A possible solution would be adding a language file that users can use to translate the prompts.

**Comment by YOUSY0US3F** [comment]
Yeah, the issue is the prompts are in English. There is already a prompt override for the main prompt But I think the behind the scenes prompts for the other instances are still in English. Did you try overriding the prompt in the config?

**Comment by GolfinhoVoador** [comment]
> Yeah, the issue is the prompts are in English. There is already a prompt override for the main prompt But I think the behind the scenes prompts for the other instances are still in English. Did you try overriding the prompt in the config?

I would try it, but I don't know what the default prompt is so I couldn't properly translate it, and I didn't manage to find it in the code either

__

Also, a bit unrelated but, It seems that sometimes o4 simply doesn't like to punish players too much, we only managed to get the AI mad enough to tp everyone into the void or flood the world with lava twice, in another life we were doing the exact contrary of what the AI asked and we didn't get any harsh punishments other than our house becoming obsidian, being deleted or being regenerated as diamond blocks for some reason

**Comment by florantis** [comment]
@GolfinhoVoador I had the same issue but I have found the prompts in the files (`src/main/java/net/bigyous/gptgodmc/GPT/Prompts.java`), it goes like this:

For Sandbox mode: 
> "You will roleplay as the god of a small minecraft island world, give individual players challenges to test their merit and reward them if they succeed. You will recieve information about what has happened on the island. Only use tool calls, other responses will be ignored."

For Deathmatch mode: 
> "You will roleplay as the god of a small minecraft world, the players are split into two teams that must fight to the death. Each team spawns on their own floating island. You will give the teams challenges to complete and reward the teams that succeed. Only use tool calls, other responses will be ignored."
